### DISCUSSION TITLE
无任务脚本基础如何启动

### DISCUSSION THREAD

**Discussion by sppi8** [mikumifa/biliTickerBuy#108] [提问专区]
![微信图片_20240613202625](https://github.com/mikumifa/biliTickerBuy/assets/172604067/3251e0b5-6403-4266-867f-9d48b2c3e00b)
如图，上面程序已全部点击一边，不知是因未设置默认打开软件导致，教程中的网址不知是哪

**Comment by mikumifa** [comment]
https://github.com/mikumifa/biliTickerBuy/wiki/%E5%B0%8F%E7%99%BD%E4%B8%8B%E8%BD%BD%E6%8C%87%E5%8D%97

**Reply by sppi8** [reply]
下是会下，没有网址来启动程序

**Comment by mikumifa** [comment]
压缩包打开里面有exe文件，打开它，会跳出一个命令行界面，里面有网址

**Reply by sppi8** [reply]
没有exe文件，所以我才问的，上面的图就是压缩包里的全部文件

**Comment by mikumifa** [comment]
仔细看文章，下载的是bilibili-buy.zip

**Comment by Allroha** [comment]
笑死
